galata/test/jupyterlab/menus.test.ts

```diff
@@ -7,6 +7,7 @@ import type { ISettingRegistry } from '@jupyterlab/settingregistry';
 const menuPaths = [
   'File',
   'File>New',
+  'File>Workspaces…',
```

Comment:
This should be `File>Workspaces` once corrected (see earlier)

Reply:
Done in 2872d004058e0751e026ced4651d14bbe07fb5b9.